09-valve-monitor/web-app/src/styles/ValveMonitorTable.module.scss

```diff
@@ -153,6 +153,12 @@
       margin: 0 0 0 16px;
     }
 
+    :global .ant-table-tbody .editable-input-name,
+    :global .ant-table-tbody .editable-button-name {
+      max-width: 155px;
+      width: 155px;
+    }
```

Comment:
Does this need to be in here twice (once outside of a media query, once inside it) to override Ant? If yes, 👍 If no, shouldn’t it be added to the NF10 CSS also?

Reply:
The columns displayed are slightly fewer for NF9 - no valve related data to show there's no need to slightly narrow the width as the table shrinks to tablet size like there is for NF9. Good call out though